src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -474,21 +474,20 @@ class EmojiPickerMenu extends Component {
                 style={[styles.emojiPickerContainer, StyleUtils.getEmojiPickerStyle(this.props.isSmallScreenWidth)]}
                 pointerEvents={this.state.arePointerEventsDisabled ? 'none' : 'auto'}
             >
-                {!this.props.isSmallScreenWidth && (
-                    <View style={[styles.ph4, styles.pb1, styles.pt2]}>
-                        <TextInput
-                            label={this.props.translate('common.search')}
-                            onChangeText={this.filterEmojis}
-                            defaultValue=""
-                            ref={el => this.searchInput = el}
-                            autoFocus
-                            selectTextOnFocus={this.state.selectTextOnFocus}
-                            onSelectionChange={this.onSelectionChange}
-                            onFocus={() => this.setState({isFocused: true, highlightedIndex: -1, isUsingKeyboardMovement: false})}
-                            onBlur={() => this.setState({isFocused: false})}
-                        />
-                    </View>
-                )}
+                <View style={[styles.ph4, styles.pb1]}>
+                    <TextInput
+                        label={this.props.translate('common.search')}
+                        onChangeText={this.filterEmojis}
+                        defaultValue=""
+                        ref={el => this.searchInput = el}
+                        autoFocus={!this.isMobileLandscape()}
+                        selectTextOnFocus={this.state.selectTextOnFocus}
+                        onSelectionChange={this.onSelectionChange}
+                        onFocus={() => this.setState({isFocused: true, highlightedIndex: -1, isUsingKeyboardMovement: false})}
+                        onBlur={() => this.setState({isFocused: false})}
+                        autoCorrect={false}
+                    />
+                </View>
                 {!isFiltered && (
```

Comment:
@stitesExpensify If you're good with[ using absolute positioning](https://github.com/Expensify/App/pull/14652#discussion_r1095972455 ) for the FlatList with the search items, I can remove the isFiltered option.

Reply:
I commented on the linked thread. I think we should only use one flatlist, so we should keep this

Reply:
I want to mention that in the case of a single flatlist, we would need to rerender the entire flatlist, including all headers, each time we clear the input. Additionally, in this case, the scroll position of the main emoji list (after tapping on some category) would not be saved after clearing the search input.
Are you sure?

Reply:
@stitesExpensify combine this?

Reply:
To clarify, are you saying that when you clear the search we would scroll back to the top?

Reply:
Yes, after clearing input, the scroll position would be reset to zero. You can now check it on the web, where there is one flatlist, and on mobile, there are two flatlists.

Reply:
Gotcha, yeah that's intentional. When you clear your search, you should be sent to the top so one flatlist is the intended functionality. Thanks for confirming

Reply:
Okay, I got you
I'll update the `.native` file and make one flatlist